src/components/styled/StyledPagination.tsx

```diff
@@ -0,0 +1,37 @@
+import { Pagination, PaginationProps } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import theme from "theme";
+
+const grey100 = theme.colors.grey100;
+
+const StyledPagination = styled(Pagination)<PaginationProps>({
+  color: grey100,
+
+  "& .MuiPaginationItem-root": {
+    color: grey100,
+    borderRadius: "0px",
+    border: `1px solid ${grey100}`,
```

Comment:
This value seems like a good candidate for the theme if we keep reusing it... no need to change it now, but let's keep our eyes on it